language-plutus-core/src/Language/PlutusCore/Core/Type.hs

```diff
@@ -54,6 +58,8 @@ data Kind ann
     | KindArrow ann (Kind ann) (Kind ann)
     deriving (Show, Functor, Generic, NFData, Lift, Hashable)
 
+$(deriveEnumerable ''Kind)
```

Comment:
I think just stick this in the test file to generate an orphan, so `language-plutus-core` doesn't have to depend on this.